test/kubeclient_test.rb

```diff
@@ -1,7 +1,4 @@
-require 'minitest/autorun'
-require 'json'
-require 'webmock/minitest'
-require './lib/kubeclient'
+require './test/test_helper'
```

Comment:
> Yeah, it should default properly... See https://github.com/ruby/ruby/blob/v2_0_0_643/lib/rake/testtask.rb#L90

found the problem...  with the default rakefile configuration, test files names must start with "test", in kubeclient they ended with "test", hence renaming solved the problem...